### ISSUE TITLE
A simple way to get rid of semicolons?

### ISSUE THREAD

**Issue by rjsteinert** [yeoman/generator-backbone#355]
Hi there - I know this is a sensitive issue but I was just hoping there might be an easy way to get rid of semicolons. I don't really need them on the teams that I work on.

**Comment by arthurvr** [comment]
No, there isn't. Every official generator agreed on one style guide, and that's the one we follow. If you integrate something like `jscs` you can easily just edit its config and use `--fix` to make all code adhere to your personal/team's style.

**Comment by rjsteinert** [comment]
Thank you for the tip! In `.jshintrc` I set `"asi": false,` so that `grunt`
would not fail or complain about semi-colons and then in `.jscsrc` I set
`"dissallowSemicolons": true,`. Unfortunately with the
`dissallowSemicolons` option, it is an exception in that it does not work
with `jscs --fix`. Oh well. My situation is already vastly improved!

- RJ

Senior Front End Engineer at Isovera, Javascript Ninja, Farm Hacker, Unix
Philosophy for IoT Evangelist
rj@rjsteinert.com
http://rjsteinert.com
http://twitter.com/rjsteinert
http://github.com/rjsteinert
